rest_framework_bulk/drf3/mixins.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import print_function, unicode_literals
+import traceback
+from rest_framework import status
+from rest_framework.mixins import CreateModelMixin
+from rest_framework.response import Response
+
+
+__all__ = [
+    'BulkCreateModelMixin',
+    'BulkDestroyModelMixin',
+    'BulkUpdateModelMixin',
+]
+
+
+class BulkCreateModelMixin(CreateModelMixin):
+    """
+    Either create a single or many model instances in bulk by using the
+    Serializer's ``many=True`` ability from Django REST >= 2.2.5.
+
+    .. note::
+        This mixin uses the same method to create model instances
+        as ``CreateModelMixin`` because both non-bulk and bulk
+        requests will use ``POST`` request method.
+    """
+
+    def create(self, request, *args, **kwargs):
+        bulk = isinstance(request.data, list)
+
+        if not bulk:
+            return super(BulkCreateModelMixin, self).create(request, *args, **kwargs)
+
+        else:
+            serializer = self.get_serializer(data=request.data, many=True)
+            serializer.is_valid(raise_exception=True)
+            self.perform_create(serializer)
```

Comment:
And `ListSerializer.create` just calls `OriginalSerializer.create` on each of the objects.

If you aren't going to support DRF 2 and DRF 3 concurrently, I'd just create a `perform_bulk_create` hook (that just calls `serializer.save`) and leave it at that.

If you want to support them concurrently, this would be the place to split away from `ListSerializer.create` and handle creation on your own by re-creating the loop that it usually uses and calling your own hooks instead.